### ISSUE TITLE
Can't draw a mask on Inpainting tab of Gradio Demo

### ISSUE THREAD

**Issue by edtjulien** [salesforce/UniControl#18]
Hello!

Thanks for this great work. I got some good results.

Btw, I would like to try the inpainting with your Gradio Demo in HF. But it seems it's no visual tools to create the mask on the original image? How to try inpainting in Gradio HF?

**Comment by canqin001** [comment]
Good idea! We can easily enable rectangular mask on this gradio demo. But the interactive masking with irregular contours would be super challenging. 

Please let me know if you have noticed similar demos that have interactive masking.

**Comment by edtjulien** [comment]
Fortunately it seems to be an option for this in Gradio Image object:

https://www.gradio.app/docs/image

The option is tool=["sketch"] and the result is a dictionary with "image" and "mask" keys
